static/src/stylesheets/module/experiments/_acquisitions-epic-design-variations.scss

```diff
@@ -0,0 +1,71 @@
+@import "svg";
+
+// Classes used by multiple variants
+// =================================
+
+.contributions__title--epic--large {
+    @include font-size(36px);
+    @include mq(tablet) {
+        @include font-size(38px);
+    }
+}
+
+// 1. extra_paragraph variant
+// ==========================
+
+// copy changes only
+
+// 2. large hook variant
+// =====================
+
+// uses large title
+
+// 3. subtle variant
+// =================
+
+.contributions__epic--subtle {
+    // Unset border top as it will be set for the inline header element
+    border-top: 0;
+    background-color: white;
+    margin-top: 50px;
+    margin-bottom: 50px;
+}
+
+.contributions__title--epic--subtle {
+    display: inline;
+    border-top: 1px solid $multimedia-main-2;
+    @include fs-header(3);
+    padding-top: $gs-baseline / 3;
+}
+
+.contributions__paragraph--subtle {
+    margin-top: 0;
+}
+
+// 4. prominent variant
+// ====================
+
+// uses large title
+
+.contributions__epic--prominent {
+    background-color: #ffd964;
```

Comment:
It possible somehow, I'd rather use `$multimedia-main-2` here. It's a bit more dull though, but imho works better together with the buttons. We also try not to introduce new colors at all :)